polling_stations/templates/address_select.html

```diff
@@ -1,40 +1,41 @@
 {% extends "base.html" %}
 
 {% load i18n %}
+{% load dc_forms %}
 
 {% block content %}
 
-<div class="row">
-  <div class="columns large-12">
-    <div class="card">
-      <h2 id="address_picker">
-        {% trans "Choose Your Address / Street" %}
-      </h2>
-
-      <form method=post class="form form-inline">
-        {% csrf_token %}
-        <div class="form-group">
-            {{ form.as_p }}
-            <button type="submit" class="button" id="submit-address">
-              {% trans "Find my Polling Station" %}
-            </button>
-        </div>
-      </form>
-
-    </div>
-
-  </div>
+<div class="card">
+  <h2 id="address_picker">
+    {% trans "Choose Your Address / Street" %}
+  </h2>
+
+  <form method=post class="form form-inline">
+    {% csrf_token %}
+    {{ form|dc_form }}
+    <button type="submit" class="button" id="submit-address">
+      {% trans "Find my Polling Station" %}
+    </button>
+  </form>
 </div>
 
+
 {% endblock content %}
 
 {% block in_page_javascript %}
 <script type="text/javascript">
-$(document).ready(function() {
-    var isAndroid = /(android)/i.test(navigator.userAgent);
-    if (isAndroid) {
-        $('.select_multirow').css('-webkit-appearance', 'menulist');
-    }
-});
+  var isAndroid = /(android)/i.test(navigator.userAgent);
+  if (isAndroid) {
+    var elements = document.querySelectorAll('.select_multirow');
+    var className = 'android_menu_list';
+    Array.prototype.forEach.call(elements, function(el, i){
```

Comment:
I know you've got to go back to Android <= 4.1 and IE9 for browsers that don't support `forEach()` but do you reckon it would be better to use a more backwards-compatible syntax here?